tableview.moon

```diff
@@ -1,11 +1,14 @@
 mxml    = require "moonxml"
 loadkit = require "loadkit"
 inspect = require "inspect"
+html = mxml.html\buffered!
 
 moonxmlFind = loadkit.make_loader "lua"
 
 generate = (tbl) ->
-  template = mxml.html\loadluafile moonxmlFind "tableview.template"
-  return template tbl, inspect
+  html.buffer\truncate(0)
+  template = html\loadluafile moonxmlFind "tableview.template"
+  template tbl, inspect
+  html.buffer\concat
```

Comment:
Isn't a `!` missing here?

Reply:
FFS this is why I hate switching between ruby and moonscript